js/src/personal/kazi/Kazi.page.tsx

```diff
@@ -29,6 +42,18 @@ export function KaziPage() {
       >
         Hey! I'm Kazi.
       </Text>
+
+      <button
```

Comment:
Should use the Mantine Button component instead of the html one.

Reply:
Just changed to using a Mantine button